SLHCUpgradeSimulations/Configuration/python/phase1TkCustoms.py

```diff
@@ -250,29 +193,6 @@ def customise_Validation_Trackingonly(process):
     return process
 
 def customise_harvesting(process):
-    # FIXME
-    #
-    # These should be added back once they get fixed
-    #
-    # The customizations are done here instead of in the central files
-    # with era because they are temporary
-
-    # Tracking DQM needs to be migrated for phase1
-    process.DQMHarvestTracking.remove(process.TrackingOfflineDQMClient)
-
-    # No HLT yet for 2017, so no need to run the DQM (avoiding excessive printouts)
-    process.DQMOffline_SecondStep.remove(process.HLTMonitoringClient)
-    process.DQMOffline_SecondStep_PrePOG.remove(process.hltOfflineDQMClient)
-    process.hltpostvalidation = cms.Sequence()
-    process.crt_dqmoffline.remove(process.dqmOfflineTriggerCert)
-
-    # Excessive printouts because 2017 doesn't have HLT yet
-    process.DQMOffline_SecondStep_PrePOG.remove(process.efficienciesRealData)
-    process.DQMOffline_SecondStep_PrePOG.remove(process.efficienciesRealElectronsData)
-    process.DQMOffline_SecondStep_PrePOG.remove(process.efficienciesRealMuonsData)
-    process.DQMOffline_SecondStep_PrePOG.remove(process.normalizePlotsRealMuonsData)
-    process.postValidation.remove(process.runTauEff)
-

```

Comment:
@makortel I had to add `process.dqmHarvesting.remove(process.sipixelCertification)` here, since said module is still present (I could not find where it comes from).

IIRC, the issue is that it depends on histograms output by the old harvesting, which I removed/replaced since it is largely useless with the new DQM.